### ISSUE TITLE
How to make pop-up like on the range-field example?

### ISSUE THREAD

**Issue by trusktr** [Dogfalo/materialize#6384]
I'm looking at the range-field example here: https://materializecss.com/range.html

In the second example, "HTML5 Range", there's a little tooltip that follows the slider. 

When I tried pasting that HTML into my app, I don't have that tooltip. Do I need JS for that? Which JS and what call do I need to make for that? 

Thanks!

**Comment by anchit1501** [comment]
can you shrare your HTML file snippet

**Comment by acburst** [comment]
Yes it requires our javascript files. Many things don't work if you don't include all the necessary files for the framework

**Comment by anchit1501** [comment]
Include materialize.min.js that should do the work

**Comment by trusktr** [comment]
@anchit1501 Hello, I pasted this:

```html
  <form action="#">
    <p class="range-field">
      <input type="range" id="test5" min="0" max="100" />
    </p>
  </form>
```

And the slider works, but there's no tooltip.

I have materialize CSS in place. Do I also need the JS in place? (I'm using modules, do I need to import something an initialize something?).

**Comment by anchit1501** [comment]
You can import  jquery and materialize.min.css, that will do the work.
https://codepen.io/anchit1501/pen/vwqWWj

**Comment by trusktr** [comment]
Hmm, I do have jQuery and the CSS in place. Do I need to call something in case I am dynamically making the DOM later (f.e. with React, Vue, Angular, etc)?

**Comment by anchit1501** [comment]
This is working for me

`import React from 'react';
import  'materialize-css/dist/js/materialize.min.js';
import 'materialize-css/dist/css/materialize.min.css';
import M from 'materialize-css';

class App extends React.Component {
  componentDidMount(){
    M.Range.init(this.range);
  }
  render(){
  return (
    <div className="App">
      <form action="#">
    <p class="range-field">
      <input  ref={(range)=>{this.range = range}} type="range" id="test5" min="0" max="100" />
    </p>
  </form>
    </div>
  );
}
}
export default App;
`

**Comment by anchit1501** [comment]
Try this approach

```
import React from 'react';
import  'materialize-css/dist/js/materialize.min.js';
import 'materialize-css/dist/css/materialize.min.css';
import M from 'materialize-css';

class App extends React.Component {
  componentDidMount(){
    M.Range.init(this.range);
  }
  render(){
  return (
    <div className="App">
      <form action="#">
           <p class="range-field">
               <input  ref={(range)=>{this.range = range}} type="range" id="test5" min="0" max="100" />
           </p>
      </form>
    </div>
  );
}
}
export default App;
```

**Comment by trusktr** [comment]
Ah, okay, so `M.Range.init(elementRef)` is the magic key. I didn't see this mentioned in the docs. I'll give it a go...
